python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -396,23 +396,41 @@ def uniform_neighbor_sample(
     else:
         indices_t = numpy.int32
 
-    if input_graph.renumbered:
-        start_list = input_graph.lookup_internal_vertex_id(start_list)
-
-    start_list = start_list.rename(start_col_name).to_frame()
+    start_list = start_list.rename(start_col_name)
     if batch_id_list is not None:
-        ddf = start_list.join(batch_id_list.rename(batch_col_name))
+        batch_id_list = batch_id_list.rename(batch_col_name)
+        if hasattr(start_list, "compute"):
```

Comment:
It looks like the docstring for `start_list` might be incomplete, since it looks like `start_list` can be an int, list, cudf.Series, or dask_cudf.Series?

Reply:
I'll update it

Reply:
done